src/applications/find-va-forms/helpers/index.js

```diff
@@ -25,10 +24,12 @@ export const normalizeFormsForTable = forms =>
       // Overridden form values.
       id,
       type: form.type,
-      lastRevisionOn: lastRevisionOn.unix(),
+      lastRevisionOn: lastRevisionOn && lastRevisionOn.unix(),
```

Comment:
Can we do `lastRevisionOn?.unix()` here?

Reply:
Works, thanks Kelson!